frontend/packages/kubevirt-plugin/src/k8s/wrapper/common/k8s-resource-wrapper.ts

```diff
@@ -79,4 +80,22 @@ export abstract class K8sResourceWrapper<
     }
     return (this as any) as SELF;
   };
+
```

Comment:
we can add this function to K8sResourceObjectWithTypePropertyWrapper as well for consistency